proto/agent.proto

```diff
@@ -47,6 +47,14 @@ message CollectorConnections {
 	// Message batching metadata
 	int32 groupId = 5;
 	int32 groupSize = 6;
+
+	// post-resolution field; all the hosts referenced in `connections` keyed by
+	// hostId
+	map<int32, Host> connectedHosts = 7;
```

Comment:
`resolvedHosts`? Can/should we also include the above top-level `Host` in this map?

Reply:
`hostsById`?

Reply:
I prefer `resolvedHosts`. `hostsById` is sort of redundant since the type explains it. I'll also rename `connectedContainers` 

> Can/should we also include the above top-level Host in this map?

that host **should** be referenced in the connections list,  but we can make it explicit that it should be included.